qcodes/instrument_drivers/Keysight/keysightb1500/KeysightB1500_module.py

```diff
@@ -300,3 +317,22 @@ def clear_timer_count(self) -> None:
         (FMT3 and FMT4).
         """
         self.root_instrument.clear_timer_count(chnum=self.channels)
+
+
+class StatusMixin:
+    def __init__(self):
```

Comment:
Since this init method has no arguments you need to add an explicit `-> None` return type to have this checked by mypy

Reply:
Thanks. i wil do that and fix if there any mypy issues.

Reply:
Done